src/pipelines/epidemiology/ly_humdata.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict
+import math
+from pandas import DataFrame
+from lib.data_source import DataSource
+from lib.time import datetime_isoformat
+import  datetime
+
+class LibyaHumdataDataSource(DataSource):
+   
+    def parse_dataframes(
+        self, dataframes: Dict[str, DataFrame], aux: Dict[str, DataFrame], **parse_opts
+    ) -> DataFrame:
+
+        # Rename the appropriate columns
+        data = (
+            dataframes[0]
+            .rename(
+                columns={
+                    "Location": "subregion1_name",
+                    "Confirmed Cases": "total_confirmed",
+                    "Deaths": "total_deceased",
+                    "Recoveries": "total_recovered",
+                    "Date": "date",
+                }
+            )
+            .drop(columns=["Active"])
+            .drop(columns=["total_deceased"])
+        )
+
+        # The first row is metadata info about column names - discard it
+        data = data[data.subregion1_name != '#loc+name']
+
+        # Convert date to ISO format
+        data["date"] = data["date"].apply(lambda x: datetime.datetime.strptime(x, "%Y-%m-%d").strftime('%d-%m-%Y'))
```

Comment:
We have a handy method to simplify this common operation: `lib.time.datetime_isoformat`

Reply:
Yep I'd tried that one, but it wasn't converting %Y-%m-%d to %d-%m-%Y.  Since it's a one off case for Libya which requires this date time conversion, I didn't feel. the. need to add. this. support. in. datetime_isoformat.

Reply:
I'm confused by your comment. The output of `datetime_isoformat` is always `%Y-%m-%d` and the parameter allows you to enter whatever date format the source of data has.

All date outputs have to be in the format of YYYY-MM-DD for the rest of the pipeline to process them.

Reply:
My bad, the Libya data is already in %Y-%m-%d so no special conversion needed then. :)

http://screen/31QXKTVe98R